tests/integration/exec.bats

```diff
@@ -323,14 +323,14 @@ function check_exec_debug() {
 	[ "$status" -eq 0 ]
 }
 
-@test "RUNC_DMZ=legacy runc exec [execve error]" {
+@test "RUNC_DMZ=false runc exec [execve error]" {
```

Comment:
So, why setting `RUNC_DMZ=false` here if this is the default?

Reply:
IMHO it seems better to be explicit when the test needs that. If we switch the default later to enabled, it will be painful to realize which tests we need to add the `RUNC_DMZ=false`.

Not because of failing tests, those are easy fix (although pointless overhead, imho), but cases exactly like this one are the tricky ones, that we want to test it with and without runc_dmz. When we switch the default, we need to find these cases where now we need to disable it. Otherwise test will just pass and we won't be testing that case.

But that is my personal preference. I've updated to remove it, to match yours :)

Reply:
That mistake is very simple to make, I was doing it here :see_no_evil: 

I've fixed it, but if you decide you prefer the explicit false/true, I have it ready in another branch locally.

Let me know what you think